packages/components/src/components/textarea/textarea.constants.ts

```diff
@@ -0,0 +1,29 @@
+import utils from '../../utils/tag-name';
+import { TYPE as FONT_TYPE } from '../text/text.constants';
+import { BUTTON_VARIANTS, ICON_BUTTON_SIZES } from '../button/button.constants';
+
+const TAG_NAME = utils.constructTagName('textarea');
+
+const WRAP = {
+  HARD: 'hard',
+  SOFT: 'soft',
+} as const;
+
+const AUTO_COMPLETE = {
+  OFF: 'off',
+  ON: 'on',
+} as const;
+
+const DEFAULTS = {
+  ICON_SIZE_VALUE: 1,
+  ICON_SIZE_UNIT: 'rem',
+  CLEAR_BUTTON_ICON: 'cancel-bold',
```

Comment:
`const ICON_NAME: Extract<IconNames, 'cancel-bold'> = 'cancel-bold';`

Then, use 
```suggestion
  CLEAR_BUTTON_ICON,
```

`const CLEAR_BUTTON_ICON: Extract<IconNames, 'cancel-bold'> = 'cancel-bold';`

Reply:
Done